tests/e2e/idempotency/test_idempotency_dynamodb.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+import pytest
+
+from tests.e2e.utils import data_fetcher
+
+
+@pytest.fixture
+def basic_handler_fn(infrastructure: dict) -> str:
+    return infrastructure.get("BasicHandler", "")
+
+
+@pytest.fixture
+def basic_handler_fn_arn(infrastructure: dict) -> str:
+    return infrastructure.get("BasicHandlerArn", "")
+
+
+def test_basic_idempotency_record(basic_handler_fn_arn: str, basic_handler_fn: str):
+    # GIVEN
+    function_name = "basic_handler.lambda_handler"
+    table_name = "IdempotencyTable"
```

Comment:
Can we get this from the CloudFormation output?

```suggestion

@pytest.fixture
def idempotency_table_name(infrastructure: dict) -> str:
    return infrastructure.get("DynamoDBTable", "")

def test_basic_idempotency_record(basic_handler_fn_arn: str, basic_handler_fn: str, idempotency_table_name: str):
    # GIVEN
    function_name = "basic_handler.lambda_handler"
    table_name = idempotency_table_name
```

Reply:
Done